apps/dashboard/test/models/user_configuration_test.rb

```diff
@@ -192,22 +193,6 @@ class UserConfigurationTest < ActiveSupport::TestCase
     assert_equal "default_value_argument", target.send(:fetch, nil, "default_value_argument")
   end
 
-  test 'USER_PROPERTIES should have correct default' do
```

Comment:
why remove this?

Reply:
In reality , it is the same test as `test "properties expected defaults" do`, so I thought it was best to delete it.

Reply:
cool